packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -147,10 +147,10 @@ export function addImplicitTags(staticGenerationStore: StaticGenerationStore) {
     }
   }
 
-  if (urlPathname) {
-    const parsedPathname = new URL(urlPathname, 'http://n').pathname
-
-    const tag = `${NEXT_CACHE_IMPLICIT_TAG_ID}${parsedPathname}`
+  // TODO: verify that this is the correct behavior
```

Comment:
What exactly do we want to verify here? feels a bit general, maybe add the thing you want to verify into the comment

Reply:
In this case, the pathname of the request (which is requester controlled) is used in the cache key. If that's not intended, then we should instead just rely on `page`.